test/parallel/test-http-should-keep-alive.js

```diff
@@ -20,10 +20,10 @@
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 'use strict';
-require('../common');
```

Comment:
I'm sorry, I should've been clearer. This is still required but it should not be assigned to a variable. Also, please try to run `make test -j4` before committing as it will flag these types of issues in advance and also run the test being modified.